CONTRIBUTING.md

```diff
@@ -6,7 +6,19 @@ Please report any issues at https://github.com/determined-ai/determined/issues.
 
 ## Contributing Changes
 
-TODO
+We welcome outside contributions. If you'd like to make a contribution, please:
+
+1. Tell us about what you'd like to contribute on
+   [our Slack](https://join.slack.com/t/determined-community/shared_invite/zt-cnj7802v-KcVbaUrIzQOwmkmY7gP0Ew)
+   or [community mailing list](https://groups.google.com/a/determined.ai/forum/#!forum/community).
+   We'd hate for you to duplicate effort is already in-flight.
+
+1. The first time you submit code, you'll need to
+   [sign a CLA](https://determined.ai/cla/).
+
+1. Submit a pull request. Someone from the Determined AI team will review the
```

Comment:
Add something about: ensuring that your changes continue to pass the linters and unit tests by running `make check` and `make test`?

Reply:
Good idea, I mentioned applying the linter with `make fmt` and running `make test`.